.travis.yml

```diff
@@ -5,10 +5,7 @@ node_js:
 env:
   global:
   - CF_API=https://api.fr.cloud.gov
-  - CF_USERNAME=gsa-18f-federalist_deployer
   - CF_ORGANIZATION=gsa-18f-federalist
```

Comment:
Interesting we're keeping the CF_ORGANIZATION here but taking it out in federalist. Shouldn't it be set in the environment here, too?